pkgs/applications/misc/kiln/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromSourcehut, scdoc }:
+
+buildGoModule rec {
+  pname = "kiln";
+  version = "0.2.0";
+
+  src = fetchFromSourcehut {
+    owner = "~adnano";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ERDStRh5M2X2Nnvut97IhN0PjZvS9ePMEscsLCiXWiI=";
+  };
+
+  nativeBuildInputs = [ scdoc ];
+
+  vendorSha256 = "sha256-bMpzebwbVHAbBtw0uuGyWd4wnM9z6tlsEQN4S/iucgk=";
+
+  doBuild = false;
```

Comment:
Please do not use the upstreams makefile logic but nixpkgs buildGoModule logic to compile.

Reply:
Upstream makefile not only builds binary, but builds man page using scdoc, install man to proper location. What is the reason to duplicate it?

Reply:
They usually assume things which do not work for NixOS.